docs/CONFIG.md

```diff
@@ -4024,6 +4057,12 @@ ResendAfterThreshold = '1m' # Default
 ```
 ResendAfterThreshold controls how long to wait before re-broadcasting a transaction that has not yet been confirmed.
 
+### LogUnconfirmedAfterThreshold<a id='EVM-Transactions-LogUnconfirmedAfterThreshold'></a>
+```toml
+LogUnconfirmedAfterThreshold = '5m' # Default
+```
+LogUnconfirmedAfterThreshold controls how long to wait for the oldest unconfirmed transaction to log critical.
```

Comment:
I wonder if we could simplify the name (and description) a little bit. I think it doesn't need to be so specific, but I'm also worried about the word 'log' having multiple meanings in this context.
Would something like `ConfirmationThreshold` make sense? And I think oldest can be left out since it is sort of implied that the oldest would be affected first.
```suggestion
ConfirmationThreshold = '5m' # Default
```
ConfirmationThreshold controls how long to wait for transaction confirmation before logging critical.
```

Reply:
I hear you on this. Adding a descriptive name was a bit difficult for this one. I chose to add the word 'log' in it just to prevent someone from thinking that this might have an actual effect on unconfirmed transactions. Same for the 'ConfirmationThreshold'. It sounds better, but I'm worried that users initially might get confused as to what exactly the config controls.

Reply:
Ah right, yeah I agree. Keeping the 'Un' prefix makes sense then too. I think there is still room to find something like `UnconfirmedXXXThreshold` though, and perhaps not say 'log' exactly.... :thinking: 
```toml
UnconfirmedAlertThreshold = '5m'
UnconfirmedCriticalThreshold = '5m'
UnconfirmedNotifyThreshold = '5m'
UnconfirmedNotificationThreshold = '5m'
```

Reply:
Oh, I like the `UnconfirmedAlertThreshold = '5m'`! Mentions both the alerting and the unconfirmed state.